### DISCUSSION TITLE
How to make Unocss breakpoints max-width instead of min-width

### DISCUSSION THREAD

**Discussion by Keshav-writes-code** [unocss/unocss#4290] [Q&A] [answered]
so,i wanted to include a class name `artboard-demo` if i am on a tablet or laptop screen, but if i am on a mobile screen, i dont want to include the class name.

this wont work because it will consider `sm` as to be the min width parameter
`sm:artboard-horizontal`
 how can i make it max width so that artboard-horizontal is appiled when the screen widh is greater than `sm`

**Comment by zyyv** [comment, marked as answer]
https://unocss.dev/presets/wind#breakpoints

Use `lt-sm` variant.
